src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/IRoslynProjectChangeProcessor.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.VisualStudio.Razor;
 
-internal interface IProjectWorkspaceStateGenerator
+internal interface IRoslynProjectChangeProcessor
```

Comment:
I highly approve of this new name! I will resist any bike-shedding, since it's so much better than the original.